runbooks/KubevirtVmHighMemoryUsage.md

```diff
@@ -0,0 +1,26 @@
+# KubevirtVmHighMemoryUsage
+
+## Meaning
+
+Container that hosting Virtual Machine has free memory less than 20 MB and it is close to memory limit.
```

Comment:
@andreyod  Hi, I think this might be better phrasing:
The container, that is hosting the Virtual Machine, has less than 20 MB free memory and is close to its memory limit.

Reply:
Changed